pypeit/edgetrace.py

```diff
@@ -4953,7 +5012,7 @@ def order_refine_free_format(self, reference_row, combined_order_tol=1.8, debug=
                                      function='legendre', lower=3., upper=3., maxiter=5,
                                      sticky=True)
 
-        # Ideally, measured widths/gaps should be regected for one of the
+        # Ideally, measured widths/gaps should be rejected for one of the
```

Comment:
Hi @kbwestfall sorry if I keep bugging you with this. 
I was looking at the ADAP runs and all completed except one that failed (which is not bad!). But as I was diagnosing the reason of the failure, I noticed that, this particular dataset, for some reason, had a ghost orders (PypeIt detected an order in the gap) as the first order (the leftmost order). Since  `trace.find_missing_orders` starts the search for missing orders from the first order, the entire search is messed up because the first order is messed up.
So, I thought we could add a condition here (similar to the `combined_orders` below) to exclude this bad order before running `trace.find_missing_orders`. It seems that it could be beneficial in general for ghost orders. So it would be to do something like:
`ghost_orders = np.append(gap, gap[-1]) / gap_fit.eval(cen) < ghost_orders_tol`

`gap` and `gap_fit.eval(cen)` have different sizes, this is why I'm appending (but I'm not sure it's the right way to deal with it). And I'm not sure how much `ghost_orders_tol` should be, but maybe 0.6 or so.

What do you think? I can send you the data if you want to take a look.

Reply:
Hi @debora-pe.  Sorry for the slow response!  Your solution sounds good to me, but I'd like to have a look to see what's happening, as well.  Can you send me the data?

Reply:
I think this should now be fixed.  See the changes in [this commit](https://github.com/pypeit/PypeIt/pull/1835/commits/08a2798c8fc21ec22dfa61b11d23751cadf0e982).